libminifi/src/utils/net/DNS.cpp

```diff
@@ -15,85 +15,20 @@
  * limitations under the License.
  */
 #include "utils/net/DNS.h"
+
+#ifndef WIN32
+#include <cstring>
+#endif /* WIN32 */
+
```

Comment:
What is this used for?

Reply:
I don't think it's used anymore, removed in 9ddaa71b8aecb723fc4c8eaaf49873a5939179f4